cpp/unreal_plugins/SpServices/Source/SpServices/EnhancedInputService.h

```diff
@@ -95,6 +104,17 @@ class EnhancedInputService : public Service {
         UnrealClassRegistrar::registerClass<UInputTriggerTap>("UInputTriggerTap");
         UnrealClassRegistrar::registerClass<UInputTriggerTimedBase>("UInputTriggerTimedBase");
 
+
+        unreal_entry_point_binder->bindFuncToExecuteOnGameThread(
+            "enhanced_input_service", "setup_enhanced_input_component", [this](uint64_t& actor) -> void {
+                APawn* pawn = toPtr<APawn>(actor);
+                SP_ASSERT(pawn);
+
+                EnhancedInputPawnWrapper* wrapper = static_cast<EnhancedInputPawnWrapper*>(pawn);
+                wrapper->CreateInputComponent(UEnhancedInputComponent::StaticClass());
```

Comment:
- Rename to `setup_player_input_component` so the name matches the C++ function being called.
- Rename to `pawn_wrapper`.
- `CreateInputComponent` is already a `UFUNCTION`, so it shouldn't be part of this entry point. Users can get the `UClass` for `UEnhancedInputComponent` themselves via existing interfaces and pass it into `CreateInputComponent` via existing interfaces.
- `InputComponent` is already a `UPROPERTY`, so users can get it themselves and pass it into this entry point.